content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -15,14 +15,7 @@ card:
 
 <!-- overview -->
 
-यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए 
-कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है।
-काटाकोडा आपके ब्राउज़र पर मुफ़्त कुबेरनेट्स वातावरण प्रदान करता है।
-
-{{< note >}}
-यदि आपने अपने स्थानीय सिस्टम पर मिनीक्यूब स्थापित किया है तो आप इस ट्यूटोरियल का अनुसरण कर सकते हैं।
-स्थापाना निर्देश के लिए [मिनीक्यूब पृष्ट](https://minikube.sigs.k8s.io/docs/start/) देखें। 
-{{< /note >}}
+यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है। ट्यूटोरियल एक कंटेनर छवि प्रदान करता है जो सभी अनुरोधों को प्रतिध्वनित करने के लिए NGINX का उपयोग करता है। 
```

Comment:
Doesn't reflect the current version of the Hello Minikube document

https://kubernetes.io/docs/tutorials/hello-minikube/.

Kindly make appropriate changes.